zschema/keys.py

```diff
@@ -168,6 +171,43 @@ def key_to_string(o):
         else:
             return o.to_string()
 
+    @staticmethod
+    def _handle_validation(policy, e):
```

Comment:
This should probably get a better name in retrospect.

Reply:
If it was go, you could define a `policy.validation_failure(e)`; but as it stands, `_handle_validation_failure(policy, e)` would make its intent clearer IMO.

Reply:
✔️